### PR TITLE
Second blog post of Team fusion

### PR DESCRIPTION
**Pull request by kalpanibhagya**

### CODE REVIEW

**Comment by Janakshi28** [comment]
Request for review : @alicetragedy

**Comment by Janakshi28** [comment]
Hi @nerdyvaishali , is there any changes that we have do with this PR?

File: blog/_posts/2017-09-14-a-song-like-summer.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+---
+title: 🎶 A Song-like Summer 🌞 
+layout: post
+created_at: Thu Sep 14 2017
```
Comment (v-thakkar) [inline blog/_posts/2017-09-14-a-song-like-summer.md]: Hi, thanks for the PR. Good job on improving many things by your own. Thanks. :) I just have 2 small comments and then blog will be ready to publish. Can you please change a date to Tue September 19, 2017 everywhere?

File: blog/_posts/2017-09-14-a-song-like-summer.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+![Coaches](/img/blog/2017/2017-09-14-coaching-sessions.jpg )
+<div class="image-credits"><b>Our coaching sessions. (Image : Kalpani’s phone, Slack and Google hangouts)</b></div>
+
+
+
+## After this summer
+
+Thanks to RGSoC, this summer is so far the best summer in our lives. And it has opened doors for, new opportunities, new friends, new way of thinking and etc. 
+
+We believe, we can do more after this summer. After RGSoC, we would love to continue our contributions to **OpenDF**. And we are even planning to participate in next year’s RGSoC. See how impressed are we from our first RGSoC experience? 😌 And we’ll be giving our support to the tech community more and more in the future, by joining meetups, conferences, competitions and engaging in many more open source projects.
```
Comment (v-thakkar) [inline blog/_posts/2017-09-14-a-song-like-summer.md]: You can't participate in RGSoC again. Can you just remove that line? :)

File: blog/_posts/2017-09-14-a-song-like-summer.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+And we must mention about the Conferences! Because we will be flying to Australia in 2018 to participate in *[RubyConf AU 2018](https://www.rubyconf.org.au/2018 )* 🛫🇦🇺 and will be participating in *[DEV DAY 2017](http://www.devday.lk/ )* 🇱🇰 conference as well. We are so excited! 😁🎉
+
+![Nothing to worry](/img/blog/2017/2017-09-14-when-things-work.gif )
+<div class="image-credits"><b>When everything works fine.. (Image : giphy.com)</b></div>
+
+Last but not the least, we would like to say thanks to everyone for supporting us throughout this journey. And **THANK YOU RGSoC** for giving us this awesome opportunity!
+
+Follow us on Twitter at [@TeamFusionJK](https://twitter.com/TeamFusionJK), [@JanakshiDulanga](https://twitter.com/JanakshiDulanga) and [@KbKalpani](https://twitter.com/KbKalpani)
```
Comment (v-thakkar) [inline blog/_posts/2017-09-14-a-song-like-summer.md]: Put full stop(.) after 1st statement.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

**Comment by Janakshi28** [comment]
Hi @nerdyvaishali , we made the changes as you requested and added some new things to the post too.. :)

**Review by v-thakkar — APPROVED** [review]

**Comment by v-thakkar** [comment]
Thanks for the changes @Janakshi28 . :)

**Comment by Janakshi28** [comment]
Hi @nerdyvaishali ,

There's another minor typo with our blog post. Once it was published only we saw it. 
https://github.com/Janakshi28/RGSoC--Images/blob/master/Screenshot%20(38).png

The sentences in this snap should be in point form, but there are not due to some error with the markdown usage. Can we send another PR with the fix for that?

**Comment by alicetragedy** [comment]
@Janakshi28 yes, please do send another PR with the fix — however make sure your local fork and version is synced before you do that, otherwise you might encounter some merge conflicts that will need fixing. Thanks in advance!

**Comment by v-thakkar** [comment]
Thanks for noticing @Janakshi28 . It's a markdown space issue and I missed it. Sure, send a PR and we will merge it.

**Comment by Janakshi28** [comment]
@nerdyvaishali , I sent a new PR for this Fix #422 .

Thank you :)

**Pull request merged**